src/app/common-components/n64-box/n64-box.component.html

```diff
@@ -1,7 +1,7 @@
 <section>
   <ng-container *ngIf="!simple">
     <header>
-      <div class="box slot">File {{ number }}</div>
+      <div class="box slot"><div class="filenum">File </div></div>
```

Comment:
Does this work cross component? i.e. If you have two game-progress components and a game-faq component, does this track correctly?

Reply:
![image](https://user-images.githubusercontent.com/73679967/146593736-0474ed36-3634-43b3-b373-e49c20fcf251.png)
Yes sir

Reply:
![image](https://user-images.githubusercontent.com/73679967/146593810-95364657-3439-4f51-bbc5-17ce09b3d9ab.png)